kyma-project/kyma
# Add monitoring to console component

**Description**

Changes proposed in this pull request:

- Add monitoring to console component.
- Make console core selected by default in Grafana WebApps dashboard


*How to test it:*
- Checkout pr branch
- Open _installer-cr.yaml.tpl_ and uncomment everything but _logging_ component
- Install kyma cluster on minikube
- Go to https://console.kyma.local and access the Grafana console by clicking Administration > Diagnostics > Status & Metrics in the left navigation.
- In Grafana, select "Webapps" dasboards. You should see console-core
- Navigate through https://console.kyma.local. After some seconds you should see monitoring data updated in Grafana dashboard

Please check my comment in the linked issue: https://github.com/kyma-project/kyma/issues/1767#issuecomment-478929700
